linode/manager
# feat: [M3-7164] - Update AGLB Certificates UI

## Description 📝

- Updates AGLB Certificates UI from UX feedback

## Major Changes 🔄
- Updates general routing to be more robust
- Removes `logs` from Load Balancers
- Changes `LoadBalancers` to `Load Balancers`
- Adds tabs to the Certificates tab
  - TLS Certificates (`downstream`)
  - Service Target Certificates (`ca`)
- Makes the Create Certificate Drawer have two dynamic modes
  - `Upload TLS Certificate`
  - `Upload Service Target Certificate`

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-09-22 at 11 46 47 AM](https://github.com/linode/manager/assets/115251059/e0267c66-9d90-49f3-aaed-ae2578935617) | ![Screenshot 2023-09-22 at 11 46 30 AM](https://github.com/linode/manager/assets/115251059/674282e2-a606-47f3-a643-514ae9ca5e49) |

## How to test 🧪
- Turn on the MSW
- Test any functionality on the `http://localhost:3000/loadbalancers/0/certificate` page
  - Verify general routing/deep-linking works
  - Check the Create drawer for both TLS Certificates and Service Target Certificates
